test/TestProjects/MgmtPropertyChooser/MgmtPropertyChooser.json

```diff
@@ -410,6 +410,14 @@
                         "type": "string"
                     },
                     "description": "The virtual machine zones."
+                },
+                "fakeResources": {
+                    "readOnly": true,
+                    "type":"array",
+                    "items": {
+                        "$ref": "#/definitions/Resource"
+                    },
+                    "description": "The fake resources for the virtual machine."
```

Comment:
While this is a fake example,  it mimics the real case in [Sql/RecommendedElasticPoolsDecoupled](https://github.com/Azure/azure-rest-api-specs/blob/50a94ccd66f49aed78efa24ceefd0d5c1186c8db/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/recommendedElasticPoolsDecoupled.json#L231-L238).